### DISCUSSION TITLE
Please help me! Github deleted my link!

### DISCUSSION THREAD

**Discussion by JumbleJamJam** [community/community#22038] [New to GitHub]
<p>So I have this link: <a href="https://raw.githubusercontent.com/Roblox/SCP/Red/Lake/Everything/DuperGUI/And/Admin%22" rel="noopener nofollow ugc">https://raw.githubusercontent.com/Roblox/SCP/Red/Lake/Everything/DuperGUI/And/Admin"</a><br>
But when I go to it it says ERROR 404 and when I try to restore it it says “Error 2367 You Cant Restore This Deleted Link” Can someone please help me IS there I way I can download a zip file of this deleted rep to get my file back!!!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If a repository has been deleted its data isn’t supposed to be available any more. Repositories can be restored under certain circumstances, but that’d need to be done by the owner of the repository (or organization owning the repository).</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5814919/181068420-05c3d1d4-7be4-4064-bdf7-55a970eb8705.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">Restoring a deleted repository - GitHub Docs</a></h3>

  <p>You can restore some deleted repositories to recover their contents.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
